### PR TITLE
Added Delilah to Hardware offload

### PR DESCRIPTION
**Pull request by niclashedam**
Delilah is the first publicly described computational storage device running eBPF. I believe it is worth mentioning.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [Comprehensive XDP offload---Handling the edge cases](https://www.netdevconf.org/2.2/session.html?viljoen-xdpoffload-talk) - An update on the topic above.
 - [hBPF - eBPF in hardware](https://github.com/rprinz08/hBPF) - An eBPF CPU written for FPGAs.
 - [OpenCSD eBPF SSD offloading](https://github.com/Dantali0n/qemu-csd) - Computational Storage simulation (QEMU) platform with FUSE LFS filesystem for Zoned Namespaces NVMe SSDs using uBPF for compute kernel offloading, all in userspace.
+- [Delilah: eBPF-offload on Computational Storage](https://www.researchgate.net/publication/370818368_Delilah_eBPF-offload_on_Computational_Storage) - The first public description of an actual computational storage device supporting eBPF-based code offload.
```
Comment (qmonnet) [inline readme.md:166]: I'm not sure whether “first” and “public” make a lot of sense to the reader in the description of the project. Although I do understand you are (rightfully) proud of it, I'd keep it terse and just tell what this is. Also what does “actual” mean in the description?

**Review by qmonnet — COMMENTED** [review]
Thanks! Would you have a direct link to the PDF, rather than linking to ResearchGate? Or even better, if there a GitHub repo that we could link to (I don't mind having the link to the paper as well)?

File: readme.md (hunk 1 above)
Comment (niclashedam) [reply readme.md:166]: With _actual_, I mean it is not _virtualised_ or _emulated_. This is the only device that supports eBPF offload to storage on a physical device.

How does this sound:
_Delilah is a Computational Storage Processor (CSP) built for eBPF offload to storage devices._

**Comment by niclashedam** [comment]
> Thanks! Would you have a direct link to the PDF, rather than linking to ResearchGate? Or even better, if there a GitHub repo that we could link to (I don't mind having the link to the paper as well)?

This should be a direct link to the ResearchGate PDF: https://www.researchgate.net/profile/Niclas-Hedam/publication/370818368_Delilah_eBPF-offload_on_Computational_Storage/links/6464a6b29533894cac7662d3/Delilah-eBPF-offload-on-Computational-Storage.pdf
As for the GitHub, it can be found here: https://github.com/delilah-csp/delilah

Note that documentation efforts are ongoing, so there is limited documentation in the GitHub repository. The PDF serves as the documentation for now.

File: readme.md (hunk 1 above)
Comment (qmonnet) [reply readme.md:166]: Thanks for the clarification! I'd use “physical” in that case, I find it more explicit than “actual”.

Anyway, yes your revised version looks good!

**Comment by qmonnet** [comment]
If you plan to add sufficient docs to the Git repo, I'd favour using that as the main link (and keeping the PDF link in the description). If not, I'd rather have the PDF link over ResearchGate, unless you've got a strong opinion about it.

Please update as necessary :slightly_smiling_face:

**Comment by niclashedam** [comment]
At the moment, the extent of the documentation and the timeline are not clear, and I believe it's better to have the PDF link to avoid having to go back and update the entry.

The text and link have been updated.

**Review by qmonnet — APPROVED** [review]

**Pull request merged**